### ISSUE TITLE
能否提供一下预训练权重

### ISSUE THREAD

**Issue by YoRha19990213** [Li-Haoqing/IRSTD-Diff#3]
我使用文章规定的数据集和训练设置上完全无法达到文章中的效果

**Comment by Li-Haoqing** [comment]
代码版本弄错了（现已更正）。这一版只是DDPM，没有监督损失。
需要在gaussian_diffusion.py的training_losses_segmentation里面：terms["mse_diff"] = mean_flat((target - model_output) ** 2)
            terms["loss_cal"] = criterion(cal, res)
            terms["mse"] = (terms["mse_diff"] + terms["mse_cal"]) / 2.
            if "vb" in terms:
                terms["loss"] = terms["mse_diff"] + terms["vb"]
            else:
                terms["loss"] = terms["mse_diff"]，

以及train_util.py的forward_backward里面： loss = (losses["loss"] * weights + losses['loss_cal'] * 10).mean()
            # loss = (losses["loss"] * weights).mean()   
这两个位置改损失及weights
